apps/language_server/lib/language_server/server.ex

```diff
@@ -789,4 +790,14 @@ defmodule ElixirLS.LanguageServer.Server do
   defp set_project_dir(state, _) do
     state
   end
+
+  defp create_gitignore(%{project_dir: project_dir} = state) do
```

Comment:
Can you change this function to not overwrite the `.gitignore` if it already exists (and don't error in that case either). My reasoning is that if a user has overwritten the `.gitignore` for some reason then we should not overwrite their changes.